lib/docstheme/default/templates/reflection.hbs

```diff
@@ -0,0 +1,79 @@
+{{#with model}}
```

Comment:
Whoa - all these files. Seems like a lot of maintenance if/when we upgrade TS versions in the future no?

Reply:
no. this is from the typedoc generator. this is a copy of the 'default' theme... and we just can adjust like we want